routes/index.js

```diff
@@ -23,10 +23,23 @@ module.exports = function(passport){
     res.render('register.njk');
   });
 
-  router.post('/register', passport.authenticate('register', {
-    successRedirect: '/confirmation',
-    failureRedirect: '/register'
-  }));
+  router.post('/register', function(req,res,next) {
```

Comment:
Minor: the arguments for the function are all clumped together `function(req,res,next)`, perhaps adding spaces will make it look nicer and keep with coding style. For example: `function(req, res, next)`